gensim/models/keyedvectors.py

```diff
@@ -550,7 +551,7 @@ def most_similar(self, positive=None, negative=None, last=None, topn=10, restric
         else:
             if (len(positive)+len(negative))%2 != 0:
                 raise ValueError("wrong input word analogies.. one or more words are missing..")            
-            mean = matutils.unitvec(array(mean).mean(axis=0)/2).astype(REAL)+self.word_vec(last, use_norm=True)
+            mean = matutils.unitvec(2*(array(mean).mean(axis=0))).astype(REAL)+matutils.unitvec(self.word_vec(last, use_norm=True)).astype(REAL)
```

Comment:
Why this change?

Reply:
Because it was mistaken. We are computing the average of all elements considering their weight, which means that if we have 10 pairs the sum will be divided by 20 and as we want the average of the ten differences we then should multiple it by 2 (so that is will be divided by 10)